app/lib/publisher/backend.dart

```diff
@@ -335,10 +354,14 @@ api.PublisherInfo _asPublisherInfo(Publisher p) =>
 /// Loads [publisherId] and checks if [userId] is an admin of the publisher, and
 /// runs the callback [fn] with it.
 ///
-/// Throws AuthenticationException if the user is provided.
+/// Throws AuthenticationException if the user is not provided.
 /// Throws AuthorizationException if the user is not an admin for the publisher.
 Future<R> withPublisherAdmin<R>(
-    String publisherId, String userId, FutureOr<R> fn(Publisher p)) async {
+  String publisherId,
+  String userId,
+  FutureOr<R> fn(Publisher p), {
+  bool clearCache = true,
```

Comment:
Let's not mix concerns... Can't we make a `_purgeCache(String publisherId)` on the backend instead?

Reply:
I've removed this from the PR, keeping only a separate `purgePublisherCache` method.